dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -224,13 +226,34 @@ class JdbcEventStore {
 
   private final NamedParameterJdbcTemplate jdbcTemplate;
 
-  @Qualifier("dataValueJsonMapper")
   private final ObjectMapper jsonMapper;
 
   private final UserService userService;
 
   private final IdentifiableObjectManager manager;
 
+  @Autowired
```

Comment:
Should we remove lombok annotation to create a constructor?

Reply:
We only use `@Autowired` in tests. Prod code uses constructors which are then used by Springs DI